android/engine/src/main/resources/strings.properties

```diff
@@ -0,0 +1,2 @@
+name.title=Mr.
+gender.male=Male
```

Comment:
Do you think we need these in `src.main` package of engine module? I think they should be in `src.test` package.